dart/test/event_processor/deduplication_event_processor_test.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:sentry/sentry.dart';
+import 'package:sentry/src/event_processor/deduplication_event_processor.dart';
+import 'package:test/test.dart';
+
+import '../mocks.dart';
+import '../mocks/mock_transport.dart';
+
+void main() {
+  group('$DeduplicationEventProcessor', () {
+    test('deduplicates if enabled', () {
+      final sut = Fixture().getSut(true);
```

Comment:
if you need a new Fixture every time which is not the case, use a lateinit var and setup, otherwise a global Fixture that creates new Sut only